pkg/ruler/ruler.go

```diff
@@ -38,16 +38,6 @@ var (
 		Name:      "ruler_ring_check_errors_total",
 		Help:      "Number of errors that have occurred when checking the ring for ownership",
 	})
-	configUpdatesTotal = promauto.NewCounterVec(prometheus.CounterOpts{
-		Namespace: "cortex",
-		Name:      "ruler_config_updates_total",
-		Help:      "Total number of config updates triggered by a user",
-	}, []string{"user"})
-	configUpdateFailuresTotal = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
This metric has been removed (replaced by `cortex_ruler_config_last_reload_successful`) but CHANGELOG doesn't mention it. Could you mention it there, please?